### DISCUSSION TITLE
Welcome to krkrz Discussions!

### DISCUSSION THREAD

**Discussion by jin1016** [krkrz/krkrz#303] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by a-rabin** [comment]
Thank you!
I am an enthusiast of KirikiriZ project and hope to make western VN with this. 
Nice to meet you!
